<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="linePort">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>90</y>
      <width>400</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="logOutput">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>130</y>
      <width>500</width>
      <height>320</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btnStart">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>460</y>
      <width>81</width>
      <height>70</height>
     </rect>
    </property>
    <property name="text">
     <string>Start Server</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnStop">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>460</y>
      <width>81</width>
      <height>70</height>
     </rect>
    </property>
    <property name="text">
     <string>Stop Server</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>30</y>
      <width>80</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Server Status</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblStatus">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>30</y>
      <width>80</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Offline</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="HostPath">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>60</y>
      <width>400</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
